src/TensorFlow.jl

```diff
@@ -145,6 +145,11 @@ function load_python_process(;force_reload=false)
                 init()
             end
         end)
+        py_version = tf_version(kind=:python)
+        jl_version = tf_version(kind=:julia)
+        if py_version < jl_version
+            warn("Your Python TensorFlow version ($py_version) is below your Julia TensorFlow version ($jl_version). This can cause various errors. Please consider upgrading your Python version by running `pip install --upgrade tensorflow` from the command line and then restarting Julia.")
```

Comment:
One can not assume tensorflow was installed via pip.
It wound't be if it was installed via Conda.jl.
Message should either just say upgrading or give conda instructions as well